runtime/CSharp/src/Tree/Pattern/ParseTreePatternMatcher.cs

```diff
@@ -648,32 +647,34 @@ internal virtual IList<Chunk> Split(string pattern)
             // collect into chunks now
             if (ntags == 0)
             {
-                string text = Sharpen.Runtime.Substring(pattern, 0, n);
-                chunks.Add(new TextChunk(text));
+                chunks.Add(new TextChunk(pattern));
```

Comment:
Wouldn't `new TextChunk(pattern.Substring(0, n))` be faster ?

Reply:
`pattern.Substring(0, n)` is equivalent to `pattern`. I don't see the reason why it should be faster.

Reply:
Yes, I was thinking of a different scenario